src/routes/products/+page.svelte

```diff
@@ -5,12 +5,12 @@
 
   const productInfo = vermouths.map((vermouth) => ({
     dir: vermouth.name.toLowerCase().replace(' ', '-'),
-    vermouth,
+    ...vermouth,
   }))
 </script>
 
 <section class="container">
-  {#each productInfo as { dir, vermouth }, index}
+  {#each productInfo as { dir, ...vermouth }, index}
```

Comment:
no

Reply:
actually I can live with this... 

I'ts not like I would have done but in this case where link is outside I think it's ok...